components/series/stackedBar.js

```diff
@@ -88,8 +52,12 @@
 
         stackedBar.yScale = fc.utilities.property(d3.scale.linear());
 
-        stackedBar.xValueKey = fc.utilities.property('name');
+        // Implicitly dependant on the implementation of the stack layout's `out`.
+        stackedBar.y0 = fc.utilities.property(function(d) {
+            return d.y0;
+        });
 
-        return stackedBar;
+        var stackLayout = d3.layout.stack();
```

Comment:
FYI, it's considered a bad practice to have a variable declared after it's first referenced. It can cause confusing bugs and is counter-intuitive to where you'd expect a declaration to be when you first look at a code block. 

It's working because JavaScript will implicitly hoist the declaration, but not the assignment, to the top of the enclosing function.